Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,13 @@
 import React from 'react';
 import './App.css';
+ 
+import CalcDisplay from './components/DisplayComponents/CalculatorDisplay'; 
```

Comment:
using inconsistent names for your imports is generally bad practice.
i would strongly encourage you to import your components EXACTLY as the files are named, this can save you from some major headaches when debugging.

```import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay'; ```